Makefile

```diff
@@ -75,6 +75,12 @@ proto/agent/agent.pb.go: proto/agent/agent.proto
 	cat hack/go-license-header.txt proto/agent/agent.pb.go > proto/agent/agent.licensed.go
 	mv proto/agent/agent.licensed.go proto/agent/agent.pb.go
 
+	mkdir -p mocks
```

Comment:
We should put these commands as part of `test:` rather than here.

Reply:
done